<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerClass</class>
 <widget class="QMainWindow" name="ViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualisation d'images</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionFermer"/>
    <addaction name="actionSauver"/>
    <addaction name="actionSauver_sous"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRefaire"/>
    <addaction name="actionDetruire_Pile_Avant_et_Arri_re"/>
   </widget>
   <widget class="QMenu" name="menuCaract_riqtique_Images">
    <property name="title">
     <string>Caractériqtique Images</string>
    </property>
    <addaction name="actionModification_de_la_Taille"/>
   </widget>
   <widget class="QMenu" name="menuModif">
    <property name="title">
     <string>Modif</string>
    </property>
    <addaction name="actionImage_Noir_et_Blanc"/>
    <addaction name="actionInverser_Image"/>
    <addaction name="actionSeuillage"/>
    <addaction name="actionModifications_des_couleurs"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuCaract_riqtique_Images"/>
   <addaction name="menuModif"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="DWdockWidget">
   <property name="windowTitle">
    <string>Seuillage</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonValider">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelMinimum">
       <property name="text">
        <string>Minimum</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelMaximum">
       <property name="text">
        <string>Maximum</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxMinimum"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxMaximum"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionSauver">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sauver</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSauver_sous">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sauver sous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRefaire">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Refaire</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDetruire_Pile_Avant_et_Arri_re">
   <property name="text">
    <string>Detruire Pile Avant et Arrière</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionModification_de_la_Taille">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modification de la Taille</string>
   </property>
  </action>
  <action name="actionSeuillage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Seuillage</string>
   </property>
  </action>
  <action name="actionImage_Noir_et_Blanc">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image Noir et Blanc</string>
   </property>
  </action>
  <action name="actionInverser_Image">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Inverser une Image</string>
   </property>
  </action>
  <action name="actionModifications_des_couleurs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Réglages des couleurs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
